packages/scoped-custom-element-registry/src/scoped-custom-element-registry.js

```diff
@@ -206,8 +209,11 @@ if (!ShadowRoot.prototype.createElement) {
 
   // Helper to create stand-in element for each tagName registered that delegates
   // out to the registry for the given element
-  const createStandInElement = (tagName) => {
+  const createStandInElement = (tagName, formAssociated) => {
     return class ScopedCustomElementBase {
+      static get formAssociated() {
```

Comment:
If the issue with `ADVANCED` compilation is that this field is being renamed (likely because closure doesn't have knowledge of the new `formAssociated field`), then doing `static get ['formAssociated']()` should fix that.

Reply:
Doing that results in `[JSC_ILLEGAL_PROPERTY_ACCESS] Cannot do '[]' access on a struct`